docs/README.md

```diff
@@ -61,7 +61,7 @@ See also https://issues.apache.org/jira/browse/SPARK-35375.
 -->
 Run the following command from $SPARK_HOME:
 ```sh
-$ sudo pip install --upgrade -r dev/requirements.txt
+$ pip install --upgrade -r ../dev/requirements.txt
```

Comment:
It syas `Run the following command from $SPARK_HOME:` so I think `dev/requirements.txt` is a proper path.

Reply:
ok, updated